nova/virt/vmwareapi/vmops.py

```diff
@@ -1848,7 +1858,9 @@ def power_off(self, instance, timeout=0, retry_interval=0):
                                             retry_interval):
             return
 
-        vm_util.power_off_instance(self._session, instance)
+        vm_ref = vm_util.get_vm_ref(self._session, instance)
```

Comment:
Why do you need that call? `vm_util.power_off_instance()` already does it for you.